src/tail/mod.rs

```diff
@@ -35,6 +38,9 @@ impl Tail {
         tunnel_port: u16,
         metrics_port: u16,
     ) -> Result<(), failure::Error> {
+        is_cloudflared_installed()?;
+        print_startup_message(&target.name, tunnel_port, metrics_port);
```

Comment:
why move this here? if anything perhaps we should modify the message, but i'd prefer to minimize the amount of printing being done in these fns.

Reply:
i moved it because i wanted the cloudflared error message to appear before we printed the "starting tunnel" message, and i moved it to a function because it's messy and doesn't need to be at the top level here

Reply:
hrm. i think it would be good to modify the message instead to say "starting tail session" and leave it. pulling to fn is good though. long term if we need more granular status updates we can use messaging to trigger them.

Reply:
withdrawn. multitasking today and i thought this was tunnel.rs.